src/styles/colors.css

```diff
@@ -1,23 +1,75 @@
 :root {
-  --palette-primary: #448fea;
-  --palette-primary-hover: #447bc6;
+  /* Action */
   --palette-action: #33ae10;
```

Comment:
Looks like a color without a modifier is the same as itself with `-base` postfix. 

We should simplify it and choose one or the other. I am voting for `base` since it means we keep some kinda of modifier on it.